src/Sys/InspectSeoHandler.php

```diff
@@ -0,0 +1,178 @@
+<?php
+declare(strict_types=1);
+
+namespace Triniti\Sys;
+
+use Defuse\Crypto\Crypto;
+use Gdbots\Ncr\Exception\NodeNotFound;
+use Gdbots\Ncr\Ncr;
+use Gdbots\Pbj\Message;
+use Gdbots\Pbjx\CommandHandler;
+use Gdbots\Pbjx\Pbjx;
+use Gdbots\Schemas\Pbjx\StreamId;
+use Gdbots\UriTemplate\UriTemplateService;
+use Google\Service\SearchConsole\InspectUrlIndexResponse;
+use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
+use Triniti\Schemas\Sys\Event\SeoInspectedV1;
+use Defuse\Crypto\Key;
+
+
+class InspectSeoHandler implements CommandHandler
+{
+    public static function handlesCuries(): array
+    {
+        return [
+            'triniti:sys:command:inspect-seo',
+        ];
+    }
+
+    public function __construct(
+        private readonly Ncr             $ncr,
+        private readonly Key             $key,
+        private readonly Flags           $flags,
+        private readonly LoggerInterface $logger = new NullLogger(),
+    )
+    {
+    }
+
+    public function handleCommand(Message $command, Pbjx $pbjx): void
+    {
+        $searchEngines = $command->get('search_engines', ['google']);
+        $nodeRef = $command->get('node_ref');
+
+        try {
+            $node = $this->ncr->getNode($nodeRef);
+        } catch (NodeNotFound $e) {
+            $this->logger->error('Unable to get node for search engines processing.', [
+                'exception' => $e,
+                'node_ref' => $nodeRef
+            ]);
+            return;
+        }
+
+        $retryCommand = clone $command;
+        $retryCommand->clear('search_engines');
+
+        foreach ($searchEngines as $searchEngine) {
+            $methodName = 'checkIndexStatusFor' . ucfirst($searchEngine);
+
+            if (!method_exists($this, $methodName)) {
+                continue;
+            }
+
+            $retryCommand = $this->$methodName($retryCommand, $pbjx, $node);
+        }
+
+        if (empty($retryCommand->get('search_engines'))) {
+            return;
+        }
+
+        $retries = $command->get('ctx_retries');
+        $maxRetries = $this->flags->getInt('inspect_seo_max_retries', 5);
+
+        if ($retries < $maxRetries) {
+            $retryCommand->set('ctx_retries', $retryCommand->get('ctx_retries') + 1);
```

Comment:
@digitalartcrew nit: you already got `ctx_retries` off the command and stored it, don't need to get it again